portal/templates/admin/admin_base.html

```diff
@@ -3,10 +3,10 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/jasny-bootstrap/3.1.3/js/jasny-bootstrap.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.11.1/extensions/export/bootstrap-table-export.min.js"></script>
 <script src="https://cdn.rawgit.com/myadzel/6405e60256df579eda8c/raw/e24a756e168cb82d0798685fd3069a75f191783f/alphanum.js"></script>
-<script src="//rawgit.com/hhurz/tableExport.jquery.plugin/master/tableExport.js" async></script>
-<script src="{{ url_for('static', filename='js/bootstrap-table-filter-control.js') }}"></script>
-<script src="{{ url_for('static', filename='js/bootstrap-datatables-extension.js') }}"></script>
-<script src="{{ url_for('static', filename='js/admin.js') }}"></script>
+<script src="https://cdn.jsdelivr.net/gh/hhurz/tableExport.jquery.plugin@master/tableExport.js" async></script>
```

Comment:
Can you pin this to a tag? Looks like `v1.10.1` (instead of `master`) is the latest...